apps/admin/src/app/challenge/[challengeId]/challenge-review.tsx

```diff
@@ -70,7 +68,7 @@ export function ChallengeReview({ challenge }: Props) {
           <Markdown>{challenge.description}</Markdown>
         </div>
         <div className="flex-1 rounded-l-xl rounded-r-2xl border border-zinc-300 dark:border-zinc-700">
-          <CodeEditor value={challenge.tests} />
+          {/* <CodeEditor value={challenge.tests} />   */}
```

Comment:
@ggrandi do we know why this is broken? I believe trash needs to use it so maybe we should make sure this works before merge.